platforms-serverless/azure-functions-linux/func-placeholder/index.ts

```diff
@@ -1,9 +1,28 @@
-import { PrismaClient, Prisma } from '@prisma/client'
-import { AzureFunction, Context, HttpRequest } from '@azure/functions'
-
-const client = new PrismaClient()
+import { Context, HttpRequest } from '@azure/functions'
+import { Prisma, PrismaClient } from '@prisma/client'
+const client = new PrismaClient({
+  datasources: {
+    db: {
+      url: process.env.AZURE_FUNCTIONS_LINUX_PG_URL as string,
+    },
+  },
+})
```

Comment:
This should not really work.

Reply:
Do you mean that this should not be nessasary?